lib/mixlib/install/product.rb

```diff
@@ -198,18 +185,6 @@ def lookup(key, version = :latest)
     config_file "/etc/opscode/chef-server.rb"
   end
 
-  product "chef-server-ha-provisioning" do
```

Comment:
It's not dead. It just hasn't needed a build in a while. We should get it back into current if it's been reaped out.